MagentoEcsFargate/README.md

```diff
@@ -0,0 +1,37 @@
+# Containerizing our Magento stack
+
+This directory contains only architecture and installation steps. For more details, please check the [blog] (https://medium.com/i-love-my-local-farmer-engineering-blog/xxxx) 
```

Comment:
(non-blocking): I assume we're waiting for the URL?

Reply:
Yes :)